tools/projmgr/src/ProjMgrGenerator.cpp

```diff
@@ -167,7 +167,7 @@ void ProjMgrGenerator::GenerateCprjTarget(XMLTreeElement* element, const Context
     targetOutputElement->AddAttribute("outdir", context.directories.outdir);
   }
 
-  // TODO Generate toolchain settings (warnings, debug, includes)
+  GenerateCprjOptions(element, context.cproject->target.build);
```

Comment:
@brondani I'm wondering if this is the correct build type container ?
I have been thinking about the case when there is options like "optimize" in the csolution and not in the cproject
IMO this should be propagated here as well.
even more, if optimize is specified in both csolution (optimize: size) and cproject(optimize: speed), then I think "speed" should override "size" optimization

Reply:
then for the time being let's keep this as is without overriding
as you suggested, let's move this discussion to #179

Reply:
I added a comment there as suggested